pkg/util/file.go

```diff
@@ -48,3 +48,26 @@ func WriteFileToLog(logger *capnslog.PackageLogger, path string) {
 
 	logger.Infof("Config file %s:\n%s", path, string(contents))
 }
+
+// CreateTempFile creates a temporary file and writes
+// the given contents into the file.
+// The name of the file is returned by the function.
+// It is the callers responsibility to remove the file when no
+// longer needed.
+func CreateTempFile(contents string) (string, error) {
+	tmpfile, err := ioutil.TempFile("", "rook")
+	if err != nil {
+		return "", err
```

Comment:
add a more meaningful error

Reply:
Hmm, what would be more meaningful here? The error returned by `ioutil.TempFile` seems to me to be the most appropriate..

Reply:
Typically, we try to name error even if it could be redundant with `err`, I'd do something like `fmt.Errorf("failed to create temporary file %+v, err)`.

Reply:
Makes sense?

Reply:
Ah, OK. Sure :)